examples/raw_input/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# all source files
+set(SRC RawInput.cpp)
+if(SFML_OS_IOS)
+    set(RESOURCES resources/tuffy.ttf)
+    set_source_files_properties(${RESOURCES} PROPERTIES MACOSX_PACKAGE_LOCATION Resources)
+endif()
```

Comment:
No need for this copy pasta `if(SFML_OS_IOS)` conditional when this example doesn't even get compiled on iOS.